### PR TITLE
Clarify what to do with out-of-order early data

### PR DESCRIPTION
**Pull request by martinthomson**
In response to review by @vasilvv.

### CODE REVIEW

**Comment by wtarreau** [comment]
It' indeed slightly clearer like this.

File: draft-ietf-httpbis-replay.md [hunk 1]
```diff
@@ -345,11 +345,12 @@ requests, which could result in increased load.
 
 ## Out of Order Delivery
 
-In protocols that deliver data out of order (such as QUIC {{HQ}}) early data
-can arrive after the handshake completes.  This leads to potential ambiguity
-about the status of requests and could lead to inconsistent treatment (see
+In protocols that deliver data out of order (such as QUIC {{HQ}}) early data can
+arrive after the handshake completes.  This leads to potential ambiguity about
+the status of requests and could lead to inconsistent treatment (see
```
Comment (kaduk) [inline draft-ietf-httpbis-replay.md]: Hmm, I wonder how we ended up with the gratuitous re-flowing.  (Different editors using different thresholds for which column to wrap at?)

Comment (martinthomson) [reply draft-ietf-httpbis-replay.md]: I use both emacs and vim at different times, but both are configured with the same value.  ¯\\_(ツ)_/¯

**Comment by martinthomson** [comment]
I've a new fix, based on discussion with davidben.

**Pull request closed without merging**